repos/system_upgrade/el7toel8/actors/selinux/selinuxapplycustom/tests/component_test.py

```diff
@@ -0,0 +1,100 @@
+import os
+
+import pytest
+
+from leapp.snactor.fixture import current_actor_context
+from leapp.models import SELinuxModule, SELinuxModules, SELinuxCustom, SELinuxFacts, SELinuxRequestRPMs
+from leapp.libraries.stdlib import api, run, CalledProcessError
+from leapp.reporting import Report
+
+test_modules = [
+    ["400", "mock1"],
+    ["99", "mock1"],
+    ["200", "mock1"],
+    ["400", "mock2"],
+    ["999", "mock3"],
+    ["400", "permissive_abrt_t"]
+]
+
+semanage_commands = [
+    ['fcontext', '-t', 'ganesha_var_run_t', "'/ganesha(/.*)?'"],
+    ['fcontext', '-t', 'httpd_sys_content_t', "'/web(/.*)?'"],
+    ['port', '-t', 'http_port_t', '-p', 'udp', '81']
+]
+
+
+def findModuleSemodule(semodule_lfull, name, priority):
+    for line in semodule_lfull:
+        if name in line and priority in line:
+            return line
+    return None
+
+
+def findSemanageRule(rules, rule):
+    for r in rules:
```

Comment:
Same question as about findModuleSemodule

```
def findSemanageRule(rules, rule):
    return next((r for r in rules if all(word in r for word in rule)), None)
```
Another question - is `all(word in r for word in rule)` strong enough to judge that a matching rule is found? If there are more words in r then in rule it will still produce True. Shouldn't we have 2 identical word-sets comparison here?

Reply:
Set comparison sounds better, will fix